package.json

```diff
@@ -8,21 +8,39 @@
     "url": "http://github.com/JustinTulloss/zeromq.node.git"
   },
   "dependencies": {
+    "bindings": "~1.2.1",
     "nan": "~2.1.0",
-    "bindings": "~1.2.1"
+    "node-pre-gyp": "^0.6.18"
   },
+  "bundledDependencies": ["node-pre-gyp"],
   "devDependencies": {
-    "should": "2.1.x",
+    "aws-sdk": "^2.2.26",
```

Comment:
That's just for uploads (if the binaries end up in an S3 bucket). I can rip this out of course, as I'd prefer to rely on GitHub releases like I do for other packages.